pkg/eosclient/eosgrpc/eoshttp.go

```diff
@@ -376,6 +378,9 @@ func (c *EOSHTTPClient) PUTFile(ctx context.Context, remoteuser string, auth eos
 		return err
 	}
 
+	if app != "" {
+		req.Header.Set("app", app)
```

Comment:
Best practice would be to name this head: `X-App`

Reply:
Agreed, let's see if Andreas can change it